python/server/qgsrequesthandler.sip

```diff
@@ -17,7 +17,7 @@
  * Examples of possible mechanisms are cgi Get, cgi Post, SOAP or the usage
  * as a standalone command line executable
  */
-class QgsRequestHandler  /Abstract/
+class QgsRequestHandler  / Abstract /
```

Comment:
Just curious: these extra spaces are from the `astyle` command? Or from `sipify` ?